### PR TITLE
Render original content for short xref inline inside links

### PR DESCRIPTION
**Pull request by yufeih**
Fixes https://dev.azure.com/ceapex/Engineering/_workitems/edit/87621/

@yishengjin1413

### CODE REVIEW

File: test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public void XrefTestGeneral()
 @Microsoft.Build.Tasks
 ""@""a[test](link)
 @hehe
+![img @hehe a](src ""title @hehe b"")
```
Comment (yishengjin1413) [inline test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs]: Would you also add cases like "![img @hehe a" and `![img @hehe a](src ""title @hehe b""`

Comment (yishengjin1413) [reply test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs]: Also `this **img @hehe a** haha`

Comment (yishengjin1413) [reply test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs]: @yufeih From the test case, For "![img @hehe a", the XrefInlineShortParser will not work also. Is there any way to fix this case?

**Comment by yufeih** [comment]
The trick is during render, `LinkInline` will use an `HtmlRenderer` with `EnableHtmlForInline = false`. The cost to fix it does not justify the reason, thus close this.

File: test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs (hunk 1 above)
Comment (yufeih) [reply test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs]: Fixed in Renderer

**Pull request merged**